### PR TITLE
Consume cluster scoped api

### PR DESCRIPTION
**Pull request by fromanirh**
Update all the stack to (deploy and) work with NRT API v0.0.12

### CODE REVIEW

**Comment by fromanirh** [comment]
replaces: https://github.com/k8stopologyawareschedwg/deployer/pull/66

**Comment by fromanirh** [comment]
we need to improve/extend out e2e tests, but that will require more powerful CI boxes :\

File: pkg/deployer/sched/sched.go [hunk 1]
```diff
@@ -49,16 +48,9 @@ func Deploy(log tlog.Logger, opts Options) error {
 		return err
 	}
 
-	rteMf, err := rtemanifests.GetManifests(opts.Platform, "")
```
Comment (cynepco3hahue) [inline pkg/deployer/sched/sched.go:52]: Where do we get manifests for the scheduler?

Comment (fromanirh) [reply pkg/deployer/sched/sched.go:52]: line 58. This was to configure the scheduler plugin to look in the RTE namespace when looking for the NRT objects.

Comment (cynepco3hahue) [reply pkg/deployer/sched/sched.go:52]: I see, thanks!

**Review by cynepco3hahue — APPROVED** [review]

**Pull request merged**